### PR TITLE
Replaces the RnG of electric shocks with a more sane calculation

### PR DESCRIPTION
**Pull request by BlackMajor**
TL;DR
Shock damage being an RnG value hard capped at 1MW with "jumps" in damage at certain breakpoints was weird so I smoothed out the math. Overall, damage is uncapped but does a little less damage.

Extended answer:
The old method returns burn damage to the victim in a series of five "tiers" based on the available power at the time.
These tiers roll a random with disadvantage, the thresholds in question were as follows:
Note that these values are rolled with disadvantage.
1kW to 50kw = 10-20
50kW to 100kW = 15-40
100kW to 200kW = 20-60
200kW to 1MW = 25-80
1MW+ = 50-160

I've replaced the math with a logarithmic that looks like this
burn damage = log(1.1, power) - log(1.1, (log(1.1, power)*1.5))
which results in a smoothed out damage curve with a plateau around 1MW, as follows:
1kW = 5
10kW = 24
100kW = 45
250kW = 53
1MW = 66
10MW = 88
100MW = 110
1GW = 132

### CODE REVIEW

**Review by Runa-Dacino — APPROVED** [review]
At first, hearing of this PR I was apprehensive, worried that we will end up with nerfed consequences and safer electricity.

This worry had been appeased, and even reversed with the uncapping! It gives reason to be careful with RCON set-up (shocks can be made harmless if you don't set output blindly to 250!) and to be smarter about how you power shields on SD rather than just consequence-free hotwiring things into the main.

I like these changes, and they have my support.

**Comment by BlackMajor** [comment]
I can tweak the math if you want more fine tuning to a specific curve.
The one implemented is just kinda a rough guess of fitting the curve to around where the old values lied, just without the whole.. break points thing.

Whole inspiration for doing this was realising there was effectively only three choices for engineers setting the RCON. Those being 99kW, 199kW and 250kW, at each respective breakpoint.

**Review by Heroman3003 — APPROVED** [review]
